code/controllers/subsystem/weeds.dm

```diff
@@ -54,8 +54,12 @@ SUBSYSTEM_DEF(weeds)
 	for(var/turf/T AS in creating)
 		if(MC_TICK_CHECK)
 			return
+		var/obj/alien/weeds/node/N = creating[T]
 		// Adds a bit of jitter to the spawning weeds.
-		addtimer(CALLBACK(src, .proc/create_weed, T, creating[T]), rand(1, 3 SECONDS))
+		if(prob(20) && (N.name == WEED) && !iswallturf(T) && !(locate(/obj/alien/weeds/node) in range(1, T)))
+			addtimer(CALLBACK(src, .proc/create_node, T, creating[T]), rand(8, 10 SECONDS))
+		else
+			addtimer(CALLBACK(src, .proc/create_weed, T, creating[T]), rand(1, 3 SECONDS))
```

Comment:
@BraveMole i know this is your idea, but isn't this going to cause other issues.
This will leave a whole in the area for a few seconds, which could allow the SS enough time to queue another weed to spawn (leading to another loop of nodes to spawn).

I think if you want to have this logic you'll need to just have the times be the same for both types (weeds / nodes)
That however is going to cause balance issues.

I'd be extra careful going ahead with this.